src/util.h

```diff
@@ -677,6 +677,27 @@ enum ESkillLevel
     SL_PROFESSIONAL = 3
 };
 
+inline QString SkillLevelToString ( const int iSkill )
+{
+    switch ( iSkill )
+    {
+    case SL_BEGINNER:
+        return QCoreApplication::translate ( "CServerDlg", "Beginner" );
+        break;
+
+    case SL_INTERMEDIATE:
+        return QCoreApplication::translate ( "CServerDlg", "Intermediate" );
+        break;
+
+    case SL_PROFESSIONAL:
+        return QCoreApplication::translate ( "CServerDlg", "Expert" );
+        break;
+
+    default:
+        return QString ( "" );
```

Comment:
There shouldn't be a default - all the options should be covered and "None" should be returned for SL_NOT_SET

Reply:
I will update with an SL_NOT_SET case -- but I believe having a default is prudent;  perhaps "Not defined" (untranslated).

Reply:
It is worth noting from a semantics perspective that saying "Skill -> None" can mean both "having no skill" and "having not made a selection"  

![image](https://user-images.githubusercontent.com/72955397/196526763-e7de6729-1d99-42eb-b6a7-df0986a8dc0f.png)

I'm not fighting the suggestion, just making the point that an empty string is more reflective of "not making a choice" than the word "None" is.

Reply:
It's about consistency.  If the "None" in the English UI is changed to "Not specified", for example, the JSONRPC response should change, if you're expecting to get the human-readable value.

Reply:
@pljones Sorry to join this conversation that late, but do we really want to expose the translated string here? Wouldn't this mean that the API client would need to handle different server languages...?

Reply:
If I'm writing a web site module that interacts over the JSONRPC interface, I don't want to have to worry about the API response getting translated because of the country Qt thinks the software is running in.  The API responses should always be the same _value_ for the same _meaning_.  How the web site then displays that in _its_ user interface is a different issue and outside the scope of JSONRPC.

Reply:
@Rob-NY This is still outstanding.

Reply:
Why can't we just return the int? If the API is well documented, the number should be sufficient.

Reply:
Still outstanding.